### DISCUSSION TITLE
Github support

### DISCUSSION THREAD

**Discussion by gldtn** [zed-industries/zed#6636] [Feature Requests]
Does Zed currently support github, like uploading to a repo etc..? If so, where can I find documentation on?

I have git_status set to true, but don't see anything related inside Zed

**Comment by JosephTLyons** [comment]
As of right now, we only support a few git-related things:

- displaying of the git hunk additions / deletions in the gutter
- branch switching / creation

We don't have a git panel yet where you can push and pull commits to and from a remote repository, but it is on the roadmap!
